src/api/app/models/event/relationship.rb

```diff
@@ -1,7 +1,30 @@
 module Event
   class Relationship < Base
     self.abstract_class = true
-    payload_keys :description, :who, :user, :group, :project, :package, :role
+    payload_keys :description, :when, :who, :user, :group, :project, :package, :role, :notifiable_id
     shortenable_key :description
+
+    def parameters_for_notification
+      super.merge({ notifiable_type: notifiable_type, notifiable_id: notifiable_id })
```

Comment:
I think AR is clever enough to figure all the out for itself if you just assign the object as `notifiable`...

Reply:
No, it's not working because this is a hash and then I need to branch by `notifiable_type`...

Reply:
ok